app/src/main/java/de/westnordost/streetcomplete/quests/surface/Surface.kt

```diff
@@ -23,3 +24,24 @@ enum class Surface(val item:Item<String>) {
 }
 
 fun List<Surface>.toItems() = this.map { it.item }
+
+fun pavedSurfaces(): List<Surface> {
+    return listOf(
+            ASPHALT, CONCRETE, PAVING_STONES,
+            SETT, UNHEWN_COBBLESTONE, GRASS_PAVER,
+            WOOD, METAL
+    )
+}
+
+fun unpavedSurfaces(): List<Surface> {
+    return listOf(
+            COMPACTED, FINE_GRAVEL, GRAVEL,
+            PEBBLES
+    )
+}
+
+fun groundSurfaces(): List<Surface> {
+    return listOf(
+            DIRT, GRASS, SAND
+    )
+}
```

Comment:
Why do you think these should be functions and not constants?

Reply:
I failed to consider that, moved to constants.